src/main/resources/db/migration/V1_1__InsertData.sql

```diff
@@ -10,14 +10,17 @@ VALUES ('c84936e4-2f0f-11e8-b467-0ed5f89f718b','Community Foundation Of Des Moin
 insert into project (proj_guid,org_guid,name,description,start_date,end_date) values ('c8493824-2f0f-11e8-b467-0ed5f89f718b','c849343c-2f0f-11e8-b467-0ed5f89f718b','Planting Bamboo','We are planting lots of Bamboo',now(), null);
 insert into project (proj_guid,org_guid,name,description,start_date,end_date) values ('c8493950-2f0f-11e8-b467-0ed5f89f718b','c849343c-2f0f-11e8-b467-0ed5f89f718b','Big Hug','We are giving lots of Hugs',now(), null);
 
-insert into user (user_guid,first_name,last_name,email,role) values ('c8493c34-2f0f-11e8-b467-0ed5f89f718b','Amanda','Panda','apanda@gmail.com','ROLE_DSMHACK_ADMINISTRATOR');
-insert into user (user_guid,first_name,last_name,email,role) values ('c84940da-2f0f-11e8-b467-0ed5f89f718b','Mike','Panda','mpanda@gmail.com','ROLE_VOLUNTEER');
+insert into user (user_guid,first_name,last_name,email,role) values ('c8493c34-2f0f-11e8-b467-0ed5f89f718b','Amanda','Panda','tbrobston@sourceallies.com','ROLE_DSMHACK_ADMINISTRATOR');
+insert into user (user_guid,first_name,last_name,email,role) values ('c84940da-2f0f-11e8-b467-0ed5f89f718b','Mike','Panda','anthonybrobston@live.com','ROLE_ORGANIZATION_ADMINISTRATOR');
+insert into user (user_guid,first_name,last_name,email,role) values ('c84943c4-2f0f-11e8-b467-0ed5f89f718b','Jan','Panda','anthonybrobston@gmail.com','ROLE_VOLUNTEER');
```

Comment:
I intend to remove these before we merge, this just makes it simpler for me during development and local testing.